### ISSUE TITLE
Results of ShapeNet dataset

### ISSUE THREAD

**Issue by edshkim98** [AnTao97/dgcnn.pytorch#29]
Hi, I just have a question regarding ShapeNet dataset.
Is the official result for ShapeNet dataset based on Test data or Validation data?
Thank you.

**Comment by edshkim98** [comment]
Hi, is the official result of DGCNN on ShapeNet dataset based on Validation dataset or Test dataset.
I realized there is quite a gap between the performance on validation and test datasets where the performance drops quite a lot in test dataset.
Thank you.

**Comment by edshkim98** [comment]
Sorry, I have a few more questions.
1. Do you normalize the shpenet data before feeding into the network? I don't see any normalization in the dataset class.
2. In data.py, it seems that you do not use random sampling method to sample 2048 points but just simply use indexing e.g. points[:2048]. Is this a correct way or should I suppose to use random sampling?

**Comment by antao97** [comment]
The official code of DGCNN in part segmentation is trained on both the training set and the validation set, and tested on the testing set. I follow this setting in my code.

The downloaded ShapeNet dataset is already normalized and randomly shuffled.

**Comment by edshkim98** [comment]
@AnTao97 Hi thank you for the reply. So do you mean that yours or the official DGCNN model is trained with training+validation dataset and then validated with the test dataset?
Does this also mean that the model is picked from the best performance on the test dataset?

**Comment by antao97** [comment]
Yes
